default/customizations/custom_sitreps.txt

```diff
@@ -0,0 +1,78 @@
+// The UserSitRepLabel tags and data are required if the message template (and label) is simply provided here rather than being drawn from the stringtable.
+// The data value for the UserSitRepLabel serves as the label to be used in the SitReps filters, and CustomSitrepPriority1 through CustomSitrepPriority4
+// will be at the top of the SitReps display.  As seen below, other labels are fine as well, but they cannot have any spaces if you want to use them in the SITREP_PRIORITY_ORDER
+
+effectsgroups = [
+
+
+EffectsGroup
+    scope = Capital
+    activation = Turn high = 1
+    effects =
+        GenerateSitRepMessage
+            message = "This is a sample Custom Sitrep, which is enabled by the [[tech SPY_CUSTOM_ADVISORIES]] and all the busy empire servants reporting information back to %system%"
+            icon = "icons/tech/categories/spy.png"
+            parameters = [
+                tag = "UserSitRepLabel" data = "CustomSitrepPriority1"
```

Comment:
Seems like label should be a separate parameter in all cases.

It seems like it shouldn't be a tag, as it's not put into the text... I suppose it doesn't have to be that way, but it seems rather kludgy to use a tag that way.